src/main/java/faang/school/accountservice/enums/OwnerType.java

```diff
@@ -0,0 +1,6 @@
+package faang.school.accountservice.enums;
+
+public enum OwnerType {
+    USER,
+    PROJECT;
+}
```

Comment:
You could put this in properties as well, ENUM isn't as handy as a prop file

Reply:
Discussed on Sunday Call. This Enum does not need any changes.